feat(docs): Adding example of managing the flux ssh keypair in terraform. (#647)
fluxcd/terraform-provider-flux: examples/github-self-managed-ssh-keypair/README.md

# GitHub via SSH

The example demonstrates how to bootstrap a KinD cluster with Flux using a GitHub repository via SSH.

> However, this time the SSH keypair is managed by Terraform. So it can be rotated as needed.

Reviewer: ```suggestion
The SSH keypair is managed by Terraform and can be rotated as needed.
```
